.github/workflows/release-please.yml

```diff
@@ -4,7 +4,7 @@ name: release-please
 on:
   push:
     branches:
-      - "main"
+      - 'main'
```

Comment:
style: i think we should decide if we use quotes or not. I think given the yaml spec it would make sense to not use any quotes, unless there is an explicit syntax that may confuse parsers (just like in docker-compose files, for instance `image: redis` but  `port: - "1234:1234"`). WDYT?